docker-compose.yml

```diff
@@ -26,35 +26,39 @@ services:
     build:
       context: .
       dockerfile: docker/Dockerfile-api
+      args:
+        - BASE_IMAGE
+        - DOCKER_GIT_CREDENTIALS
     environment:
-      - AUTHENTICATION
-      - DEBUG=true
-      - DOMAINS
-      - IPS
-      - FQDN
-      - GUNICORN_LOG_LEVEL
       - ADMIN_EMAIL
       - ADMIN_PASSWORD
-      - POSTGRES_PASSWORD
-      - SECRET_KEY
+      - AUTHENTICATION
+      - CALLBACK_AUTHENTICATION
+      - CALLBACK_SSL_VERIFICATION
+      - CLIENT_ID
+      - CLIENT_SECRET
+      - DEBUG
       - DOCKER_TAG
+      - DOMAINS
+      - FQDN
       - GIT_BRANCH
+      - GUNICORN_LOG_LEVEL
       - IN_DOCKER=1
+      - IPS
+      - MAX_DISK_USAGE
+      - MAX_TASK_RESULTS
```

Comment:
Shouldn't MAX_TASK_RESULTS be removed?

Reply:
Good catch, I missed this when merging master into this branch